server/src/fetch-legal-document.ts

```diff
@@ -19,9 +29,10 @@ export default async function fetchLegalDocument(
   if (textHTML && fetchedAt > Date.now() - CACHE_AGE) {
     return textHTML;
   }
+  const legalLocale = localeMapping[locale] ? localeMapping[locale] : locale;
```

Comment:
Absolutely not a blocker, but FYI Typescript now supports [Nullish Coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing)!

```typescript
// Before:
  const legalLocale = localeMapping[locale] ? localeMapping[locale] : locale;

// Functionally equivalent:
  const legalLocale = localeMapping[locale] || locale;

// Only falls back to default if localeMapping[locale] is null or undefined:
  const legalLocale = localeMapping[locale] ?? locale;
```

Reply:
Very cool! Pushed the change to master directly, thanks for the flag.